packages/runtime/container-runtime/src/connectionTelemetry.ts

```diff
@@ -162,6 +194,7 @@ class OpPerfTelemetry {
                 duration,
                 category,
                 pingLatency: this.pingLatency,
+                durationInboundQueue: this.durationSittingInInboundQueue ?? 0,
```

Comment:
Assert above ensures it's never undefined here.
But if we had such case, we should use undefined, not zero. Undefined will result in empty field and it's distinguishable from other values, whereas zero is a valid outvome, and thus can't be used as a special value.